config/config-observability.yaml

```diff
@@ -85,19 +85,40 @@ data:
   
     # metrics.backend-destination field specifies the system metrics destination.
     # It supports either prometheus (the default) or stackdriver.
-    # Note: Using stackdriver will incur additional charges
+    # Note: Using stackdriver will incur additional charges.
     metrics.backend-destination: prometheus
 
     # metrics.request-metrics-backend-destination specifies the request metrics
     # destination. If non-empty, it enables queue proxy to send request metrics.
     # Currently supported values: prometheus, stackdriver.
     metrics.request-metrics-backend-destination: prometheus
 
-    # metrics.stackdriver-project-id field specifies the stackdriver project ID. This
-    # field is optional. When running on GCE, application default credentials will be
+    # metrics.stackdriver-project-id field specifies the stackdriver project ID.
+    # This field is optional. When running on GCE, application default credentials will be
     # used if this field is not provided.
+    #
+    # To use a Stackdriver project ID that does not match the application default credentials,
+    # add a Secret named "stackdriver-service-account-key" to the "knative-serving" namespace.
```

Comment:
I think you gave the ability to choose namespace. Did you revert that?

Reply:
The ability to choose namespace adds an extra knob with very little benefit, and possibly makes it easier to accidentally expose secrets from other namespaces based on the use of RoleBindings.